distributor-api-v1/operations.md

```diff
@@ -4,7 +4,7 @@
 
 Preferred initial call used to obtain all static data about distributor configuration for the client.
 
-### Request`[PlatformAddress]/api/distributor/v1/configuration/get` <a id="request-platformaddressapidistributorv1hotelsget"></a>
```

Comment:
Actually not sure whether I can change the anchors here as well, is it included in other docs as a link? It might so I will most probably revert this change of `id`